src/Toolbox.php

```diff
@@ -3006,6 +3006,35 @@ public static function getFgColor(string $color = "", int $offset = 40, bool $in
         return "#" . $fg_color;
     }
 
+
+    public static function isColorDark(string $color = ""): bool
+    {
+        if ($color !== "") {
+            $color = str_replace("#", "", $color);
+
+            // if transparency present, get only the color part
+            if (strlen($color) === 8 && preg_match('/^[a-fA-F0-9]+$/', $color)) {
+                $tmp = $color;
+                $alpha = hexdec(substr($tmp, 6, 2));
+                $color = substr($color, 0, 6);
+
+                if ($alpha <= 100) {
+                    return "inherit";
```

Comment:
Method is supposed to return a boolean, not a string. It will fail with a PHP error.